tests/Wallet/TestWallet.nim

```diff
@@ -0,0 +1,65 @@
+import unittest2
+
+#Util lib.
+import ../../src/lib/Util
+
+#Wallet libs.
+import ../../src/Wallet/Address
+import ../../src/Wallet/Wallet
+
+#Random standard lib.
+import random
+
+const TEST_WALLET_ITERATIONS {.intdefine.}: int = 1
```

Comment:
As a general comment, this test should not be fuzzed. There's a limited set of cases that can be tested for. That said, since it is fuzzed, please bump the iteration count to 8.